src/controller/java/BdxOTASender.cpp

```diff
@@ -73,15 +73,12 @@ CHIP_ERROR BdxOTASender::Init(System::Layer * systemLayer, Messaging::ExchangeMa
 CHIP_ERROR BdxOTASender::Shutdown()
 {
     assertChipStackLockedByCurrentThread();
-
-    VerifyOrReturnError(mSystemLayer != nullptr, CHIP_ERROR_INCORRECT_STATE);
```

Comment:
should we keep this check? It is still valid check, no?

Reply:
we don't use this member in this function, we have done check inside ResetTransfer, thanks